tests/integration/Cake.Common/ArgumentAliases.cake

```diff
@@ -0,0 +1,18 @@
+#load "./../scripts/xunit.cake"
+
+//////////////////////////////////////////////////////////////////////////////
+
+Task("Cake.Common.ArgumentAliases.HasArgument")
+    .Does(() =>
+{
+    // Given, When
+    var arg = Argument<string>("customarg");
+
+    // Then
+    Assert.Equal("hello", arg);
```

Comment:
But as I write this, I realize the Test should be called `Cake.Common.ArgumentAliases.Argument` :smile: